### PR TITLE
Transfer atqc lib

### PR DESCRIPTION
**Pull request by mistergreen22**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,16 @@
 language: python
 
 python:
-- '3.6'
```
Comment (oshtaier) [inline .travis.yml:4]: Do you have confirmation from devs or customer that we should not support these python versions for that integration?

Comment (mistergreen22) [reply .travis.yml:4]: Hi, all relay integrations we implement on python 3.7, I had talked with Kostiantyn and he agreed that we can launch tests only on python 3.7. Actually in all integrations except this and qualys we use only 3.7, launching on other versions doesn't really make sense.

Comment (oshtaier) [reply .travis.yml:4]: Actually running unit tests on multiple versions make perfect sense, so you can as usually restrict functional tests to only one version

Comment (mistergreen22) [reply .travis.yml:4]: Of course running unit test multiple versions makes sense in itself, but in our case I disagree. Please take a look at our serverless relay documentation template - https://github.com/CiscoSecurity/tr-05-serverless-relay
image
Almost all integration that we are implementing we test only on 3.7 python version(API module didn't count). So in order to achieve similarities for all integration we can test it on python 3.7 only, more of that even our lambdas using python 3.7

Comment (mistergreen22) [reply .travis.yml:4]: dev make this changes in his pr, so i will close this

**Comment by mistergreen22** [comment]
changes already merged in dev pr - https://github.com/CiscoSecurity/tr-05-serverless-microsoft-graph-security/pull/16/files

**Pull request closed without merging**